src/main/java/org/eclipse/pathfinder/api/ShortestPathAiImpl.java

```diff
@@ -0,0 +1,31 @@
+package org.eclipse.pathfinder.api;
+
+import dev.langchain4j.model.azure.AzureOpenAiChatModel;
+import dev.langchain4j.service.AiServices;
+import jakarta.enterprise.context.ApplicationScoped;
+
+@ApplicationScoped
+public class ShortestPathAiImpl implements ShortestPathAi {
+    private final ShortestPathAi shortestPathAi;
+
+    public ShortestPathAiImpl() {
+        AzureOpenAiChatModel model = AzureOpenAiChatModel.builder()
+                .apiKey(System.getenv("AZURE_OPENAI_KEY"))
+                .endpoint(System.getenv("AZURE_OPENAI_ENDPOINT"))
+                .deploymentName(System.getenv("AZURE_OPENAI_DEPLOYMENT_NAME"))
+                .temperature(0.2)
+                .logRequestsAndResponses(true)
+                .build();
+
+        // Create AI service
+        this.shortestPathAi = AiServices.builder(ShortestPathAi.class)
+                .chatLanguageModel(model)
+                .build();
+    }
```

Comment:
An `@ApplicationScoped` bean is a singleton. Therefore, I don't think we should be creating a new instance of `shortestPathAi` at each invocation. Do a `null` check and only create it the first time.